lib/modules/manager/unity3d/extract.ts

```diff
@@ -0,0 +1,55 @@
+import { regEx } from '../../../util/regex';
+import type { PackageDependency, PackageFileContent } from '../types';
+
+export const fileMatchRegex: string[] = [
+  '(^|/)ProjectSettings/ProjectVersion.txt',
+];
+
+const supportedExtensionsKey = [
+  'm_EditorVersionWithRevision',
+  'm_EditorVersion',
+];
+const parseLine = (line: string): PackageDependency | null => {
```

Comment:
```suggestion
function parseLine(line: string): PackageDependency | null {
```
No unnecessary anonymous functions.